packages/camera/src/components/Controls/FullScreenButton.js

```diff
@@ -16,7 +16,11 @@ export default function FullScreenButton() {
         textTransform: 'uppercase',
       }}
     >
-      {t('controls.fullScreen')}
+      {t(label)}
     </Text>
   );
 }
+
+FullScreenButton.propTypes = {
```

Comment:
Plz also add `FullScreenButton.defaultProps` for default value. It can be empty string here.